src/query/service/src/table_functions/async_crash_me.rs

```diff
@@ -55,6 +55,7 @@ impl AsyncCrashMeTable {
         _table_func_name: &str,
         table_id: u64,
         table_args: TableArgs,
+        _conf: &common_config::Config,
```

Comment:
we can have a global instance in common_conf ?

Reply:
like `GlobalRuntime`, `GlobalCatalog`

Reply:
Is the config immutable? If so, I think we can do this. And the context also uses this global config?

Reply:
Yes. Config never changed during runtime.

Reply:
We may have a reloadable config in the future, `GlobalConfig` can always return the latest config.

Reply:
I changed all `ctx.get_config` to `GlobalConfig::instance()` in this PR. And I removed `get_config` in context types.